rdrf/rdrf/templatetags/project_title.py

```diff
@@ -5,4 +5,7 @@
 
 @register.simple_tag
 def project_title():
-    return "%s" % (settings.PROJECT_TITLE)
+    try:
```

Comment:
You could test for attribute like this:   hasattr(settings, "PROJECT_TITLE")

Reply:
Fixed.